docker/release/config/opensearch-dashboards/opensearch-dashboards-docker-entrypoint-1.x.sh

```diff
@@ -176,7 +173,7 @@ function setupSecurityDashboardsPlugin {
 function runOpensearchDashboards {
     longopts=()
     for opensearch_dashboards_var in ${opensearch_dashboards_vars[*]}; do
-        # 'opensearch.hosts' -> 'OPENSEARCH_URL'
+        # 'opensearch.hosts' -> 'OPENSEARCH_HOSTS'
```

Comment:
`opensearch.hosts` is converted to `OPENSEARCH_HOSTS`. `OPENSEARCH_URL` is not correct.